src/backend/opencl/platform.cpp

```diff
@@ -673,22 +672,26 @@ void DeviceManager::markDeviceForInterop(const int device, const fg::Window* wHa
 #if defined(_WIN32) || defined(_MSC_VER)
                 CL_WGL_HDC_KHR, (cl_context_properties)wHandle->display(),
 #else
-                CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
+                //CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
 #endif
                 CL_CONTEXT_PLATFORM, (cl_context_properties)plat(),
```

Comment:
This is a terrible way to fix this, I know. But this seems to work without a hitch on my machine. Without removing it, it segfaults on the platform that supports opencl-opengl interop, but is not connected to the display.

@shehzan10 @syurkevi @9prady9 can you look at it as well and tell me why this may break on other machines.